package.json

```diff
@@ -101,6 +101,7 @@
     "@aws-sdk/client-chime-sdk-messaging": "^3.0.0",
     "@aws-sdk/util-hex-encoding": "^3.47.0",
     "@types/ua-parser-js": "^0.7.35",
+    "aws-sdk": "^2.1216.0",
```

Comment:
So for the demo changes we do not need `aws-sdk` dependency in the JS SDK root right?

Reply:
removed